test/unit-testing/simple-stroage-r1b3-setup.ts

```diff
@@ -159,7 +161,9 @@ describe('SimpleStorageR1B3Setup', function () {
               plugin: pluginBuild1.address,
               currentHelpers: [],
               data: ethers.utils.defaultAbiCoder.encode(
-                buildMetadata3.pluginSetupABI.prepareUpdate.fromBuild1,
+                buildMetadata3.pluginSetupABI.prepareUpdate['1'].inputs.map(
```

Comment:
Either everything should be a string or a number. See the diff block below.

Speaking of JS/TS, it's probably better to rely on strings, since objects indexed by numbers may resemble an array and open the door to unexpected edge cases

Reply:
fae5fab68683ad7b63caa38dc164d959b3d92577